code/scripting/lua.cpp

```diff
@@ -271,6 +271,11 @@ void script_state::OutputLuaMeta(FILE *fp)
 	ade_table_entry *ate;
 	fputs("<dl>\n", fp);
 
+	//***Version and copyright info
+	fprintf(fp, "<dd>%s</dd>\n", LUA_RELEASE);
+	fprintf(fp, "<dd>%s</dd>\n", LUA_COPYRIGHT);
```

Comment:
I don't think we do need the copywrite, I originally just wanted to put in the version number, but I was under the impression from @MageKing17 that it might be a good idea to put in the copywrite and authors, too.